caravel/assets/javascripts/SqlLab/components/QueryTable.jsx

```diff
@@ -42,6 +52,11 @@ class QueryTable extends React.Component {
       if (q.endDttm) {
         q.duration = fDuration(q.startDttm, q.endDttm);
       }
+      q.userId = (
+        <a onClick={this.props.onUserClicked.bind(this, q.userId)}>
```

Comment:
when we don't have an `href` we should use a button with a link class:

`<button class="btn btn-link" onClick={}>Link</button>`

http://getbootstrap.com/css/#buttons-options